CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # Changelog
 
+## 0.4.13
+
+* Use VERSION file when generating docs.
+* 64-bit gcc on Windows build error
+* New: setZerosOnOuter, increaseRowNonZeroCountsOnOuterTo
+* Longer docstrings on new methods
+* Remove a nupic.math test that should stay in nupic
+* Use unittest, not unittest2, to match the nupic.core CI config
+* Add C++ unit tests for the new SparseMatrix methods
+* Finish moving nupic.bindings math tests to nupic.core
+* s/AboveAndBelow/BelowAndAbove
+* Unit tests for new SparseMatrix methods
+* Fix changes due to RNG change in nupic.core.
+* Changed all copyright headers on all files to AGPL.
+* Removes StdRandom class that was defined in math.i but never used.
+* Tests folder refactored
+* Update unit test for swigged shuffle function.
+* Remove assertSequenceEqual that doesn't work with 2.6
+* Python 2.6 fix
```

Comment:
eh?  I think this is an unintended consequence of replaying git histories.  I don't think this release adds any meaningful support for 2.6